packages/web-app-files/tests/unit/helpers/textUtils.spec.js

```diff
@@ -0,0 +1,7 @@
+describe('textUtils', () => {
+  // textUtils compares two strings to provide a natural sort
+  // say `a` is the first string and `b` is the second
+  it.todo('should return negative integer if "b" comes before "a"')
+  it.todo('should return positive integer if "a" comes before "b"')
```

Comment:
I would prefer this to test some more variations, e.g.
- "aa" vs "ab"
- "abb" vs "aba"
- "0" vs "1"
- ...

Reply:
added some variations for each cases

Reply:
actually `.each` cannot be joined with the `.todo` method so I've commented the variations out